### PR TITLE
Exposed fgbio options --async-io, --compression, and --tmp-dir

### PR DESCRIPTION
**Pull request by nh13**

### CODE REVIEW

File: tasks/src/main/scala/dagr/tasks/fgbio/FgBioTask.scala [hunk 1]
```diff
@@ -43,7 +44,9 @@ object FgBioTask {
   *
   * @param compressionLevel the compress level to use for HTSJDK.
   */
-abstract class FgBioTask(var compressionLevel: Option[Int] = None)
+abstract class FgBioTask(var compressionLevel: Option[Int] = None,
+                         val asyncIo: Option[Boolean] = None,
+                         val tmpDir: Option[DirPath] = None)
```
Comment (tfenne) [inline tasks/src/main/scala/dagr/tasks/fgbio/FgBioTask.scala]: Want to add `logLevel` while you're at it?

File: tasks/src/main/scala/dagr/tasks/fgbio/FgBioTask.scala [hunk 2]
```diff
@@ -55,8 +58,10 @@ abstract class FgBioTask(var compressionLevel: Option[Int] = None)
   override final def args: Seq[Any] = {
     val buffer = ListBuffer[Any]()
     val jvmProps = mutable.Map[String,String]()
-    compressionLevel.foreach(c => jvmProps("samjdk.compression_level") = c.toString)
     buffer.appendAll(jarArgs(this.fgBioJar, jvmProperties=jvmProps, jvmMemory=this.resources.memory))
+    asyncIo.foreach(a => buffer ++= Seq("--async-io", a))
```
Comment (tfenne) [inline tasks/src/main/scala/dagr/tasks/fgbio/FgBioTask.scala]: Why not `asyncIo.foreach(a => buffer.append("--async-io", a))`?

**Review by tfenne — APPROVED** [review]
A couple of minor comments.

**Comment by tfenne** [comment]
Also builds are failing because `GroupReadsByUmi` defines a `tmpDir` parameter without an `@override`.

File: tasks/src/main/scala/dagr/tasks/fgbio/FgBioTask.scala (hunk 1 above)
Comment (nh13) [reply tasks/src/main/scala/dagr/tasks/fgbio/FgBioTask.scala]: Want to submit a PR to my PR?  Joking.

**Pull request merged**